### DISCUSSION TITLE
初来乍到

### DISCUSSION THREAD

**Discussion by yixiaomoya** [share121/inter-knot#1956] [其他]
#芝士逸筱墨(这照片是我.jpg)
![qq_pic_merged_1722768173235](https://github.com/user-attachments/assets/53c5362b-697d-4c71-8f56-49c0a931d2d3)
  
##以及,我打算不定期在这里发些自己没事写的打油诗或是随便写的乱七八糟的文章用于存档(
今天先放上一篇我挺早之前写的一段
  
云深路暗渐踱步,初见青帘慢化幕.
蔽履提携走石歧,耳旁声无仅雨瀑.
想效先者寒悟豁,只觉流水作风没.
行态稍倾猛觉错,溯游本就无所浊.
  
(ps:是谁闲的没事想独自一人感受雨中散步的浪漫?不带伞的那种.
显然是我)

**Comment by cpu644** [comment]
大佬

**Reply by yixiaomoya** [reply]
怎会被称之为大佬口牙,当不起,
我写的都是很中二的东西(

**Comment by lllliufeng** [comment]
我承认，当初是我把yixiaomoya老师赶出文坛的（

**Reply by yixiaomoya** [reply]
岂可修,名字后面把那个ya打出来很尴尬的咩!

**Comment by AnbulesPeo** [comment]
呱！是大佬口牙！
